Assets/LeapMotion/Core/Scripts/Utils/HandUtils.cs

```diff
@@ -482,6 +482,30 @@ public static class HandUtils {
                                          / deltaTime;
       }
     }
+    
+    #region Frame Utils
+
+    public static Hand Get(this Frame frame, Chirality whichHand) {
+      return frame.Hands.Query().FirstOrDefault(h => h.IsLeft == (whichHand == Chirality.Left));
+    }
+
+    #endregion
+
+    #region Provider Utils
+
+    public static Hand Get(this LeapProvider provider, Chirality whichHand) {
+      List<Hand> hands;
+      if (Time.inFixedTimeStep) {
+        hands = provider.CurrentFixedFrame.Hands;
+      }
+      else {
+        hands = provider.CurrentFrame.Hands;
+      }
+
+      return hands.Query().FirstOrDefault(h => h.IsLeft == (whichHand == Chirality.Left));
```

Comment:
Could refactor this to use the extension method above, which targets a Frame object, to reduce code duplication

Reply:
refactore'd